mcode/rems-smart-on-fhir
# Send hook depending on medication & handle multiple REMS admins with patient-view hook

## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

**Changes**
* Added REMS admin lookup table and lookup logic from request-generator
* Renamed some variables
* Updated commit hash pointed to in `src/cds-hooks`

**Related PRs**
* https://github.com/mcode/rems-cds-hooks/pull/18
* https://github.com/mcode/rems-setup/pull/40

## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

[REMS-660](https://jira.mitre.org/browse/REMS-660) and [REMS-678](https://jira.mitre.org/browse/REMS-678)

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you. 

# How to manually test this
1. Use `dev` for all repos, except src/cds-hooks mentioned above
2. Go to request-generator, authenticate, click "Launch SMART on FHIR app"
3. Open network tab
4. Note that 4 network requests were made to rems-patient-view and 4 responses were returned back
5. If the Turalio Patient Enrollment Form has been completed, you should see 4 Patient Status Update Form cards (is this correct?)
6. Otherwise, you will not see any cards
